Server/src/genericInterfaces/ResponsesInterface.ts

```diff
@@ -0,0 +1,19 @@
+export interface IResponse {
+    status: string
+    statusCode: number
+    response?: string
+}
+
+export interface SuccessIResponse extends IResponse {
+    status: "Success",
+    statusCode: 200,
+    adminRights: string;
+}
+
+export interface FailureIResponse extends IResponse {
+    status: "Failure"
+    statusCode: 400;
+}
+
+export interface ForbiddenIResponse extends IResponse {
```

Comment:
I see you are putting in the const values in the interface, imo the better approach is to `export const ForbiddenResponse: IResponse {
   statusCode: 400,
   ...
}`

Reply:
Above

Reply:
this was really more referring to the fact how you instantiate an interface